src/pages/kyubinkim/Login/Login.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react";
+import "../Login/Login.scss";
+
+class LoginKyubin extends Component {
+  state = {
+    userName: "",
+    userPass: "",
+    infoChecked: false,
+    passChecked: false,
+    buttonColor: "#bedffc",
+    move: false,
+  };
+
+  // Main으로 가기
+  goToMain = () => {
+    if (this.state.move) {
+      this.props.history.push("/Main-kyubin");
+    }
+  };
+
+  //input @ 체크
+  infoInputCheck = (e) => {
+    this.setState({ userName: e.target.value });
+    if (e.target.value.includes("@")) {
+      this.setState({ infoChecked: true }, () => this.color());
+    } else {
+      this.setState({ infoChecked: false }, () => this.color());
+    }
+  };
```

Comment:
```suggestion
  handleIdValue = (e) => {
    this.setState({ userName: e.target.value }, () => this.infoInputCheck());
  }
  
  infoInputCheck = () => {
    const isValidUserName = this.state.userName.includes("@");
     this.setState({ infoChecked: isValidUserName }, () => this.color());
  };
```

- 함수는 기능별로 분류해주시기 바랍니다.
- boolean 데이터를 활용하면 더욱 편리합니다.
- 굳이 불필요한 프로퍼티를 state에 정의해줄 필요 없겠네요.
- 함수형 컴포넌트 사용하는 것보다 기초적인 내용이 훨씬 중요하고 아직 해야 할게 많습니다. 이전에 배운 내용이 있으셔도 커리큘럼에 따라서 기본적인 것들 잘 정리하시고 다음 스텝으로 넘어가주시기 바랍니다.

Reply:
네 알겠습니다~~